### ISSUE TITLE
Edge2face experiment with CelebA-HQ 

### ISSUE THREAD

**Issue by Code-Author** [NVIDIA/pix2pixHD#336]
Can you  post how you train the Pix2Pix HD model to learn Edge 2 face ?  Like what ngf and ndf did you use ?   What batch sizes? learning rate?  How long did it take for your model to learn?  What size were you using ?   I have been  training a Pix2Pix HD model  with both ngf and ndf set to 128 for 7 days now some images are close where other way off.  I am doing a size of 1028 X 1028.   I trying to see what I need to do to get better outputs.

here is my logfile:
https://docs.google.com/spreadsheets/d/1iaYbW5LLdJMvTSfX_D38V6H_2s9NI8QYywD2VNUkb9Q/edit?usp=sharing

so any help you be great
